src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -193,13 +208,66 @@ public static boolean checkTree(@NotNull IBlockAccess world, BlockPos pos)
             return false;
         }
 
-        //Get base log, should already be base log
-        BlockPos basePos = getBaseLog(world, pos);
+        final BlockPos[] baseAndTOp = getBottomAndTopLog(world, pos, new LinkedList<>(), null, null);
+
+        if(baseAndTOp == null || baseAndTOp.length < 2)
```

Comment:
getBottomAndTop log never returns null or anything that isn't length 2. So we don't need to do this runtime check.